packages/peregrine/lib/talons/RootComponents/Product/productDetailFragment.gql.js

```diff
@@ -14,7 +14,10 @@ export const ProductDetailsFragment = gql`
         }
         id
         media_gallery_entries {
```

Comment:
`media_gallery_entries`  is DEPRECATED Use product's `media_gallery` instead
https://devdocs.magento.com/guides/v2.4/graphql/interfaces/product-interface.html?itm_source=devdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=